packages/create-mud/scripts/copy-templates.sh

```diff
@@ -11,6 +11,7 @@ find ./dist/templates/* -name "package.json" -type f | while read -r file; do
   cat "$file"
   # GPT-4 recommended perl to edit-in-place rather than sed, because sed wasn't working on CI
   perl -pi -e 's|"(?=@latticexyz)([^"]+)":\s*"link:[^"]+"|"\1": "{{mud-version}}"|g' "$file"
+  perl -pi -e 's|"(solhint-plugin-mud)":\s*"file:[^"]+"|"\1": "{{mud-version}}"|g' "$file"
```

Comment:
just to confirm this works as expected, you can run `pnpm build` in `create-mud` and look at `dist/templates` to see if the solhint plugin version got swapped with the latest canary?

Reply:
![Capture](https://github.com/latticexyz/mud/assets/62725447/01237b1e-4586-41e9-a1c0-3b0044d77b49)